civiwiki/api/urls_templates.py

```diff
@@ -2,6 +2,7 @@
 import views_templates as v
 
 urlpatterns = [
+    url(r'^createpage', v.create_page, name='add page'),
```

Comment:
It's for reverse lookup if we ever have a namespace conflict from different apps, doubt we will get to that case so now it's just a descriptor